sdk/synapse/Azure.Analytics.Synapse.Spark/src/Generated/Models/SparkStatementLanguageType.cs

```diff
@@ -23,16 +23,16 @@ public SparkStatementLanguageType(string value)
         }
 
         private const string SparkValue = "spark";
-        private const string PysparkValue = "pyspark";
-        private const string DotnetsparkValue = "dotnetspark";
+        private const string PySparkValue = "pyspark";
+        private const string DotnetSparkValue = "dotnetspark";
         private const string SqlValue = "sql";
 
         /// <summary> spark. </summary>
         public static SparkStatementLanguageType Spark { get; } = new SparkStatementLanguageType(SparkValue);
         /// <summary> pyspark. </summary>
-        public static SparkStatementLanguageType Pyspark { get; } = new SparkStatementLanguageType(PysparkValue);
+        public static SparkStatementLanguageType PySpark { get; } = new SparkStatementLanguageType(PySparkValue);
         /// <summary> dotnetspark. </summary>
-        public static SparkStatementLanguageType Dotnetspark { get; } = new SparkStatementLanguageType(DotnetsparkValue);
+        public static SparkStatementLanguageType DotnetSpark { get; } = new SparkStatementLanguageType(DotnetSparkValue);
```

Comment:
I would have guessed you'd capitalize this `DotNetSpark` based on "✔️ DO use PascalCasing for all public member, type, and namespace names consisting of multiple words," but I don't know for sure what the precedent is for dot net - do you know?

Reply:
We have more DotNet than Dotnet in our source code (https://gist.github.com/chamons/915db1c7efdc7400b4aaff68287eeeb1) but very few are in non-strings/namespaces.

This is generated code, and defined here:

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/data-plane/Microsoft.Synapse/preview/2019-11-01-preview/sparkJob.json#L1449

Reply:
I'd err on the side of being consistent with the customization layers of other Track 2 libraries, rather than relying on what you get from autogenerated code.  You can rename properties with [code generator transforms](https://github.com/Azure/autorest.csharp#rename-a-model-property) if that makes sense for this library.